_posts/2023-10-21-beneath-the-shell.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Beneath the shell
+author: Tommaso Becca
+---
+
+The history of the Ka'het (and their older cousins, the Ka'sei) has always been rather obscure, especially regarding the earlier steps that led to their creation - often leading to misunderstandings regarding the hows and whys behind their construction. It is time to examine the way these two interesting species came to be.
+
+<img class="centered" src="/images/blog/ka-het/header.jpg" width="840" height="350" />
+
+## Early concepts: the "super-sizing"
+
+The story begins over two centuries before the first ka'het saw the light; not when the proposal was first conceived (the idea itself is much older, and always rejected as outlandish and unfeasible), but when it was first realized, through a tech demo that applied what is known as "super-sizing" an organism and created a three meter long, half a meter large worm, incapable of sustaining its own weight in a gravitational field or survive on its own. Though its appearance would be much different from what followed, and it had no functions beyond eating and growing, nothing would have been possible without the breakthroughs of the super-sizing project - as a matter of fact, it wasn't long after this that the Fetri'sei program began to make its first steps.
+
+<img class="centered" src="/images/blog/ka-het/super-sized.jpg" width="400" height="300" />
+
+## The Fetri'sei, and the exploration of deep space
+
+It was around this time that, between the various departements of the interstellar authority, the Ka'sei were nearing their peak: managing over two dozen stations in the core systems (with three more in various stages of construction) and a number smaller projects on the surface, they had grown far beyond the role they were supposed the have and made many inside the government unease about the power they could exert. To start breaking the egemony that was starting to form the Ka'mar were created, intended to take some of the most promising space-based projects and develop them to completion - for instance, the then-new program of automating a large part of the interstellar traffic through new, robotic spacecrafts and surface bases. The Ka'sei's response didn't have to wait long: a competing program, which instead of automated spacecrafts put super-sized organisms at the helm and would have been capable of space travel, exploration and self replication; thus, the Fetri were born.
```

Comment:
```suggestion
It was around this time that, between the various departments of the interstellar authority, the Ka'sei were nearing their peak: managing over two dozen stations in the core systems (with three more in various stages of construction) and a number smaller projects on the surface, they had grown far beyond the role they were supposed the have and made many inside the government unease about the power they could exert. To start breaking the hegemony that was starting to form the Ka'mar were created, intended to take some of the most promising space-based projects and develop them to completion - for instance, the then-new program of automating a large part of the interstellar traffic through new, robotic spacecrafts and surface bases. The Ka'sei's response didn't have to wait long: a competing program, which instead of automated spacecrafts put super-sized organisms at the helm and would have been capable of space travel, exploration and self replication; thus, the Fetri were born.
```